containers/mysql/8.0.18/.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
+!workspace
```

Comment:
This exclusion is not necessary.

Reply:
Thanks, deleted.
https://github.com/cybozu-go/moco/pull/533/commits/2c2a3e4c5329364229e2ab5edf64c1dda7b65f1d